Duckling/Time/EN/Rules.hs

```diff
@@ -2484,27 +2484,27 @@ ruleDurationInWithinAfter = Rule
       _ -> Nothing
   }
 
-ruleDurationLastNext :: Rule
-ruleDurationLastNext = Rule
-  { name = "last|past|next <duration>"
-  , pattern =
-    [ regex "([lp]ast|next)"
-    , dimension Duration
-    ]
-  , prod = \tokens -> case tokens of
-      (Token RegexMatch (GroupMatch (match:_)):
-       Token Duration DurationData{TDuration.grain, TDuration.value}:
-       _) -> case Text.toLower match of
-         "next" -> tt $ cycleN True grain value
-         _      -> tt $ cycleN True grain (- value)
-      _ -> Nothing
-  }
+-- ruleDurationLastNext :: Rule
+-- ruleDurationLastNext = Rule
+--   { name = "last|past|next <duration>"
+--   , pattern =
+--     [ regex "([lp]ast|next)"
+--     , dimension Duration
+--     ]
+--   , prod = \tokens -> case tokens of
+--       (Token RegexMatch (GroupMatch (match:_)):
+--        Token Duration DurationData{TDuration.grain, TDuration.value}:
+--        _) -> case Text.toLower match of
+--          "next" -> tt $ cycleN True grain value
+--          _      -> tt $ cycleN True grain (- value)
+--       _ -> Nothing
+--   }
 
 ruleDurationTheLastNext :: Rule
 ruleDurationTheLastNext = Rule
   { name = "the last|past|next <duration>"
```

Comment:
Remove `the` from the rule `name` here. Change the above name to `ruleDurationLastNext` as well